x-pack/plugins/security_solution/common/endpoint/data_generators/fleet_action_generator.ts

```diff
@@ -34,7 +33,9 @@ export class FleetActionGenerator extends BaseDataGenerator {
         agents: [this.seededUUIDv4()],
         user_id: 'elastic',
         data: {
-          command: this.randomResponseActionCommand(),
+          command: overrides.data?.command
+            ? overrides.data?.command
+            : this.randomResponseActionCommand(),
```

Comment:
Was this really necessary? because `merge()` will already deeply merge values that are passed in via `overrides`

Reply:
Oh, it seems that you're right, it wasn't necessary, thanks!